leeteasy/__main__.py

```diff
@@ -31,6 +32,10 @@ def execute_start(time, difficulty, sleep_duration) -> None:
     Example: leeteasy 13:15
     """
     TimeValidator.validate(time)
+
+    if datetime.now().time() > TimeValidator.validate(time):    #To notify users of missed notifications
```

Comment:
```suggestion
    if datetime.now().time() > valid_time:
```

Reply:
I have added the requested fixes but what do I rename my branch to?
Can't seem to think of a decent name